pytest_mypy_plugins/collect.py

```diff
@@ -16,6 +16,7 @@
     Set,
 )
 
+import jsonschema
```

Comment:
It is not listed as the project's requirement, is it?

Reply:
Ooops! 🙃 Now fixed